src/test/operators/maintenance/drop_view_test.cpp

```diff
@@ -44,15 +44,32 @@ TEST_F(DropViewTest, DeepCopy) {
   EXPECT_EQ(copy->get_output(), nullptr);
 }
 
-TEST_F(DropViewTest, CanDropViews) {
+TEST_F(DropViewTest, Execute) {
   EXPECT_TRUE(StorageManager::get().has_view("view_name")) << "View not found";
 
-  auto dv = std::make_shared<DropView>("view_name");
+  auto dv = std::make_shared<DropView>("view_name", false);
   dv->execute();
 
   EXPECT_EQ(dv->get_output()->row_count(), 0u) << "DropView returned non-empty table";
 
   EXPECT_FALSE(StorageManager::get().has_view("view_name")) << "View was not removed";
 }
 
+TEST_F(DropViewTest, ExecuteWithIfExists) {
+  EXPECT_TRUE(StorageManager::get().has_view("view_name")) << "View not found";
```

Comment:
This looks like you are testing if the test was correctly setup. That is a bit confusing. Only test the things that the test is supposed to check.